enabler/test/de/schildbach/pte/live/ItalyProviderLiveTest.java

```diff
@@ -30,7 +30,7 @@
 {
 	public ItalyProviderLiveTest()
 	{
-		super(new ItalyProvider(secretProperty("navitia.authorization")));
+		super(new ItalyProvider("https://api.navitia.io/v1/", secretProperty("navitia.authorization")));
```

Comment:
This is already the default, no? Except for the s in https. If Navitia supports https, we should probably change the default.